packages/teams-js/src/public/runtime.ts

```diff
@@ -141,6 +141,10 @@ interface IRuntimeV2 extends IBaseRuntime {
       readonly mediaStream?: {};
       readonly sharedFrame?: {};
     };
+    readonly visualMedia?: {
+      readonly image?: {};
+      readonly video?: {};
+    };
```

Comment:
@jekloudaMSFT  is the desired behavior when adding a new capability to add it only to the latest runtime object? Or all previous runtime objects (since it's additive)?

Or only... v3 and v2?

Reply:
The new capability should only be added to the latest runtime object. There's technically nothing wrong with adding it to the older interfaces, but the older versions of teams-js that actually use those older interfaces will never get those interface changes (or the changes that implement the APIs that use them)

Reply:
Thanks @jekloudaMSFT @AE-MS . I updated with only add the visualMedia to v3